bin/mocks/mock-subscriptions

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+import argparse
+import random
+from datetime import timedelta
+
+from snuba.datasets.entities import EntityKey
+from snuba.datasets.entities.factory import get_entity
+from snuba.datasets.factory import get_dataset
+from snuba.datasets.table_storage import KafkaTopicSpec
+from snuba.redis import redis_client
+from snuba.subscriptions.data import PartitionId, SnQLSubscriptionData
+from snuba.subscriptions.entity_subscription import EventsSubscription
+from snuba.subscriptions.store import RedisSubscriptionDataStore
+from snuba.subscriptions.subscription import SubscriptionCreator
+from snuba.utils.metrics.timer import Timer
+from snuba.utils.streams.topics import Topic
+
+parser = argparse.ArgumentParser(description="Mock event subscriptions")
+parser.add_argument(
+    "--number", type=int, help="Number of event subscriptions to create", dest="number"
+)
+parser.add_argument(
+    "--project-id",
+    type=int,
+    help="Project IDs for subscriptions",
+    dest="project",
+    nargs="+",
+)
+
+parsed = parser.parse_args()
+
+dataset_name = "events"
+dataset = get_dataset(dataset_name)
+entity_key = EntityKey.EVENTS
+entity = get_entity(EntityKey.EVENTS)
+event_topic_spec = KafkaTopicSpec(Topic.EVENTS)
+
+timer = Timer("mock-subscriptions")
+
+for _ in range(parsed.number):
+    project_id = random.choice(parsed.project)
+
+    subscription_data = SnQLSubscriptionData(
+        query=f"MATCH (events) SELECT count() AS count WHERE project_id = {project_id}",
+        project_id=project_id,
+        time_window=timedelta(minutes=1),
+        resolution=timedelta(minutes=1),
```

Comment:
any reason not to parameterize these?

Reply:
It could be, or it could generate randomly different windows but I will not add it yet as it's not something we need right now